configurations/Rainier 1S4U Chassis.json

```diff
@@ -13,12 +13,12 @@
             "Type": "SupportedConfiguration",
             "SupportedType": "PowerSupply",
             "SupportedModel": "2B1D",
-            "RedundantCount": 4,
+            "RedundantCount": 2,
             "InputVoltage": [
                 110,
                 220
             ],
-            "PowerConfigFullLoad": true
+            "PowerConfigFullLoad": false
```

Comment:
I wonder if now that 1S4U only supports 2 power supplies, if we should set the full load property to true for both configurations so that the gpio gets set when running with 2, but maybe doesn't matter.

Reply:
Just changing the property to true would put the OCCs into over subscription with two power supplies.  We could consider logic changes between the configuration and the GPIO setting, but I'd suggest that being done in a later story.